Detectors/ZDC/simulation/src/Detector.cxx

```diff
@@ -455,7 +455,7 @@ void Detector::createMaterials()
   // ******** MATERIAL DEFINITION ********
   Mixture(0, "Walloy$", aW, zW, dW, 3, wW);
   Mixture(1, "CuZn$", aCuZn, zCuZn, dCuZn, 2, wCuZn);
-  Mixture(2, "SiO2$", aq, zq, dq, 2, wq);
+  Mixture(2, "SiO2$", aq, zq, dq, -2, wq);
```

Comment:
could you please elaborate on the reason to change sign?

Reply:
We use Geant3 convention, where positive/negative number of elements in a mixture gives a different meaning of the "wq" parameter: when positive, wq means the proportion by weights and when negative, wq is the proportions by number of atoms.
In this case, I got an exception from G4, so I checked this material definition with AliRoot, where it is defined with -2. Please, check this change with this detector experts.

Reply:
Ok thanks. @coppedis.